generators/csharp/dynamic-snippets/src/__test__/__snapshots__/DynamicSnippetsGenerator.test.ts.snap

```diff
@@ -178,109 +198,133 @@ exports[`snippets (default) > exhaustive > 'POST /container/list-of-objects (sim
 "using Acme;
 using Acme.Types.Object;
 
-var client = new AcmeClient{
-    Token = "<YOUR_API_KEY>"
-}
-await client.Endpoints.Container.GetAndReturnListOfObjectsAsync(new List<ObjectWithRequiredField>(){
-    new ObjectWithRequiredField{
-        String = "one"
-    },
-    new ObjectWithRequiredField{
-        String = "two"
-    },
-    new ObjectWithRequiredField{
-        String = "three"
-    },
-}).ConfigureAwait(false)"
+var client = new AcmeClient(
+    token: "<YOUR_API_KEY>"
+);
+
+await client.Endpoints.Container.GetAndReturnListOfObjectsAsync(
+    new List<ObjectWithRequiredField>(){
+        new ObjectWithRequiredField{
+            String = "one"
+        },
+        new ObjectWithRequiredField{
+            String = "two"
+        },
+        new ObjectWithRequiredField{
+            String = "three"
+        },
+    }
+);
+"
 `;
 
 exports[`snippets (default) > exhaustive > 'POST /container/list-of-primitives (s…' 1`] = `
 "using Acme;
 
-var client = new AcmeClient{
-    Token = "<YOUR_API_KEY>"
-}
-await client.Endpoints.Container.GetAndReturnListOfPrimitivesAsync(new List<string>(){
-    "one",
-    "two",
-    "three",
-}).ConfigureAwait(false)"
+var client = new AcmeClient(
+    token: "<YOUR_API_KEY>"
+);
+
+await client.Endpoints.Container.GetAndReturnListOfPrimitivesAsync(
+    new List<string>(){
+        "one",
+        "two",
+        "three",
+    }
+);
+"
 `;
 
 exports[`snippets (default) > file-upload > 'POST /' 1`] = `
 "using Acme;
 using Acme.Service;
 
-var client = new AcmeClient(
-    
-)
-await client.Service.PostAsync(new MyRequest(
-    
-)).ConfigureAwait(false)"
+var client = new AcmeClient();
+
+await client.Service.PostAsync(
+    new MyRequest()
+);
```

Comment:
I'm guessing this will be complicated, but it would be great if this was on one line. Maybe in the far future.

Reply:
Agreed - we can improve the formatting like this in other snippets too (e.g. Go). I'll leave this as-is for now since it's _correct_, but we can improve the style later on as we go.